scripts/start.js

```diff
@@ -347,6 +357,10 @@ function getUserInfo () {
     defaultInput: DEFAULT_DOMAIN
   });
 
+  var userId = readlineSync.question(`oVirt user id (${DEFAULT_UUID}), can be found there ${engineUrl}api/users/ : `, {
```

Comment:
The fallback must be there since the same script is used within docker environment.
So I would try to make GET call to get users and if it fails then recent prompt is used.